### PR TITLE
Fixes #69 issue

### LINKED ISSUE #69: Energi version still reflects Dash version
From debug.log we are still showing Dash versioning:

2017-09-24 14:19:19 Energi Core version v0.12.2.0-0e5bc35-dirty (2017-09-23 20:04:16 -0600)

### PR DESCRIPTION
**Pull request by levongh**
fixed issue with versioning
changed MASTERNODE_COLLATERAL_AMOUNT from int to int64_t to avoid
overflow
some coding style changes

### CODE REVIEW

**Comment by levongh** [comment]
[Energi version still reflects Dash version](https://trello.com/c/yJTYaJbk/29-energi-version-still-reflects-dash-version)

File: src/main.cpp [hunk 1]
```diff
@@ -1998,14 +1998,14 @@ void static InvalidChainFound(CBlockIndex* pindexNew)
         pindexBestInvalid = pindexNew;
 
     LogPrintf("%s: invalid block=%s  height=%d  log2_work=%.8g  date=%s\n", __func__,
-      pindexNew->GetBlockHash().ToString(), pindexNew->nHeight,
-      log(pindexNew->nChainWork.getdouble())/log(2.0), DateTimeStrFormat("%Y-%m-%d %H:%M:%S",
-      pindexNew->GetBlockTime()));
+            pindexNew->GetBlockHash().ToString(), pindexNew->nHeight,
```
Comment (ediston) [inline src/main.cpp]: Why?

File: src/main.cpp [hunk 2]
```diff
@@ -1998,14 +1998,14 @@ void static InvalidChainFound(CBlockIndex* pindexNew)
...
+            log(pindexNew->nChainWork.getdouble())/log(2.0), DateTimeStrFormat("%Y-%m-%d %H:%M:%S",
+                pindexNew->GetBlockTime()));
     CBlockIndex *tip = chainActive.Tip();
     assert (tip);
     LogPrintf("%s:  current best=%s  height=%d  log2_work=%.8g  date=%s\n", __func__,
-      tip->GetBlockHash().ToString(), chainActive.Height(), log(tip->nChainWork.getdouble())/log(2.0),
-      DateTimeStrFormat("%Y-%m-%d %H:%M:%S", tip->GetBlockTime()));
+            tip->GetBlockHash().ToString(), chainActive.Height(), log(tip->nChainWork.getdouble())/log(2.0),
```
Comment (ediston) [inline src/main.cpp]: Why?

Comment (levongh) [reply src/main.cpp]: It's just coding style changes

**Pull request merged**